pkg/collectors/builder_test.go

```diff
@@ -363,7 +365,8 @@ func TestBuilder_buildManagedClusterCollectorWithClient(t *testing.T) {
 		whiteBlackList    whiteBlackLister
 	}
 	type args struct {
-		client dynamic.Interface
+		client        dynamic.Interface
```

Comment:
do we still need dynamic client?

Reply:
It's used by getHubClusterID https://github.com/stolostron/clusterlifecycle-state-metrics/blob/main/pkg/collectors/utils.go#L34, let me create another PR to remove it.

Reply:
replace dynamic.Interface with openshift client in [commit](https://github.com/stolostron/clusterlifecycle-state-metrics/pull/67/commits/014d10ac5e489f918fa670b90417ca776b2abebb) . This PR is blocked by renaming to stolostron, will merge it once rename is done.